src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import static core.basesyntax.Lottery.getRandomBall;
+
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            System.out.println(getRandomBall().toString());
```

Comment:
Calling 'toString()' is redundant here because 'println' internally calls the 'toString()' method of the object. You can simply pass the object to 'println'.